index.html

```diff
@@ -162,5 +162,49 @@ <h2 class="my-work-title">Multi-Post Stories</h2>
         <button class="visualize-button" type="button">See Project</button>
     </div>
    </section>
+
+   <section class="my-contact-section">
+    <h2 class="contact-us">CONTACT US</h2>
+    <p class="contact-quote">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+    <form action="https://formspree.io/f/xayvgowp" method="POST">
+        <ul class="my-form">
+            <li>
+                <input type="text" name="user_name" placeholder="Text input" maxlength="30" required>
+            </li>
+            <li>
+                <input type="email" name="user_email" placeholder="aeiou@gmail.com" required>
+            </li>
+            <li>
+               <textarea name="user_name" id="msg" maxlength="500" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Good job so far, please add a `required` attribute to your `textarea` because all your input fields have to activate the validation on submit. :smiley: